MSVC compatibility

Edit: probably connected to [this](https://stackoverflow.com/questions/1372480/c-redefinition-header-files-winsock2-h)
Hello,
I have seen you guys have added CMake support.
I am trying to compile the lib on Windows using MSVC.
I am faced with cryptic errors which state that macros are trying to be redefined many times.
My command and log for cmake:
<details>

```
$> git clone <MQTT-C-repo>
$> cd <MQTT-C-repo>
$> mkdir build
$> cd build
$> cmake -G "Visual Studio 16 2019" -A Win32 -DMQTT_C_EXAMPLES=OFF ..
-- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.19041.
-- The C compiler identification is MSVC 19.26.28806.0
-- Check for working C compiler: F:/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.26.28801/bin/Hostx64/x86/cl.exe
-- Check for working C compiler: F:/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.26.28801/bin/Hostx64/x86/cl.exe - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Configuring done
-- Generating done
-- Build files have been written to: C:/Users/booko/Documents/my_github/MQTT-C/build
```

</details>

So far so good, but then, this using `cmake --build .` I am faced with the following redeclarations and redefinitions for AF_IPX etc.
<details>

```
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(103,1): warning C4005: "AF_IPX": ponowna definicja makra [C:\Users\booko\Docum
ents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(457): message : zobacz poprzednią definicję "AF_IPX" [C:\Users\booko\Documents\my 
_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(147,1): warning C4005: "AF_MAX": ponowna definicja makra [C:\Users\booko\Docum
ents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(476): message : zobacz poprzednią definicję "AF_MAX" [C:\Users\booko\Documents\my 
_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(185,1): warning C4005: "SO_DONTLINGER": ponowna definicja makra [C:\Users\book 
o\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(399): message : zobacz poprzednią definicję "SO_DONTLINGER" [C:\Users\booko\Docum 
ents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(238,25): error C2011: "sockaddr'': "struct" wpisz ponownie definicję [C:\Users 
\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(482): message : zobacz deklarację "sockaddr" [C:\Users\booko\Documents\my_github\ 
MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(440,5): error C2059: błąd składniowy: "stała" [C:\Users\booko\Documents\my_git 
hub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(575,1): warning C4005: "IN_CLASSA": ponowna definicja makra [C:\Users\booko\Do 
cuments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(284): message : zobacz poprzednią definicję "IN_CLASSA" [C:\Users\booko\Documents 
\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(581,1): warning C4005: "IN_CLASSB": ponowna definicja makra [C:\Users\booko\Do 
cuments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(290): message : zobacz poprzednią definicję "IN_CLASSB" [C:\Users\booko\Documents 
\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(587,1): warning C4005: "IN_CLASSC": ponowna definicja makra [C:\Users\booko\Do 
cuments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(296): message : zobacz poprzednią definicję "IN_CLASSC" [C:\Users\booko\Documents 
\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(598,1): warning C4005: "INADDR_ANY": ponowna definicja makra [C:\Users\booko\D 
ocuments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(301): message : zobacz poprzednią definicję "INADDR_ANY" [C:\Users\booko\Document 
s\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(600,1): warning C4005: "INADDR_BROADCAST": ponowna definicja makra [C:\Users\b 
ooko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(303): message : zobacz poprzednią definicję "INADDR_BROADCAST" [C:\Users\booko\Do 
cuments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(636,28): error C2011: "sockaddr_in'': "struct" wpisz ponownie definicję [C:\Us 
ers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(309): message : zobacz deklarację "sockaddr_in" [C:\Users\booko\Documents\my_gith 
ub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(136,23): error C2011: "fd_set'': "struct" wpisz ponownie definicję [C:\Users\boo
ko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(65): message : zobacz deklarację "fd_set" [C:\Users\booko\Documents\my_github\MQT 
T-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(156,1): warning C4005: "FD_CLR": ponowna definicja makra [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(94): message : zobacz poprzednią definicję "FD_CLR" [C:\Users\booko\Documents\my_ 
github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(171,1): warning C4005: "FD_SET": ponowna definicja makra [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(99): message : zobacz poprzednią definicję "FD_SET" [C:\Users\booko\Documents\my_
github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(180,16): error C2011: "timeval'': "struct" wpisz ponownie definicję [C:\Users\bo 
oko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(108): message : zobacz deklarację "timeval" [C:\Users\booko\Documents\my_github\M 
QTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(236,17): error C2011: "hostent'': "struct" wpisz ponownie definicję [C:\Users\bo 
oko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(164): message : zobacz deklarację "hostent" [C:\Users\booko\Documents\my_github\M 
QTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(249,16): error C2011: "netent'': "struct" wpisz ponownie definicję [C:\Users\boo 
ko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(177): message : zobacz deklarację "netent" [C:\Users\booko\Documents\my_github\MQ 
TT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(256,17): error C2011: "servent'': "struct" wpisz ponownie definicję [C:\Users\bo 
oko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(184): message : zobacz deklarację "servent" [C:\Users\booko\Documents\my_github\M 
QTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(268,18): error C2011: "protoent'': "struct" wpisz ponownie definicję [C:\Users\b 
ooko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(196): message : zobacz deklarację "protoent" [C:\Users\booko\Documents\my_github\ 
MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(364,24): error C2011: "WSAData'': "struct" wpisz ponownie definicję [C:\Users\bo 
oko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(319): message : zobacz deklarację "WSAData" [C:\Users\booko\Documents\my_github\M 
QTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(462,18): error C2011: "sockproto'': "struct" wpisz ponownie definicję [C:\Users\ 
booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(491): message : zobacz deklarację "sockproto" [C:\Users\booko\Documents\my_github 
\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(504,16): error C2011: "linger'': "struct" wpisz ponownie definicję [C:\Users\boo 
ko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(528): message : zobacz deklarację "linger" [C:\Users\booko\Documents\my_github\MQ
TT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(517,1): warning C4005: "SOMAXCONN": ponowna definicja makra [C:\Users\booko\Docu 
ments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(541): message : zobacz poprzednią definicję "SOMAXCONN" [C:\Users\booko\Documents 
\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(552,1): warning C4005: "FD_READ": ponowna definicja makra [C:\Users\booko\Docume 
nts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(559): message : zobacz poprzednią definicję "FD_READ" [C:\Users\booko\Documents\m
y_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(555,1): warning C4005: "FD_WRITE": ponowna definicja makra [C:\Users\booko\Docum 
ents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(560): message : zobacz poprzednią definicję "FD_WRITE" [C:\Users\booko\Documents\ 
my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(558,1): warning C4005: "FD_OOB": ponowna definicja makra [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(561): message : zobacz poprzednią definicję "FD_OOB" [C:\Users\booko\Documents\my 
_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(561,1): warning C4005: "FD_ACCEPT": ponowna definicja makra [C:\Users\booko\Docu 
ments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(562): message : zobacz poprzednią definicję "FD_ACCEPT" [C:\Users\booko\Documents 
\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(564,1): warning C4005: "FD_CONNECT": ponowna definicja makra [C:\Users\booko\Doc 
uments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(563): message : zobacz poprzednią definicję "FD_CONNECT" [C:\Users\booko\Document 
s\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(567,1): warning C4005: "FD_CLOSE": ponowna definicja makra [C:\Users\booko\Docum 
ents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(564): message : zobacz poprzednią definicję "FD_CLOSE" [C:\Users\booko\Documents\ 
my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1625,1): error C2375: "accept": zmiana definicji; różne inne wiązanie [C:\Users\ 
booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(739): message : zobacz deklarację "accept" [C:\Users\booko\Documents\my_github\MQ 
TT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1647,1): error C2375: "bind": zmiana definicji; różne inne wiązanie [C:\Users\bo
oko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(744): message : zobacz deklarację "bind" [C:\Users\booko\Documents\my_github\MQTT 
-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1668,1): error C2375: "closesocket": zmiana definicji; różne inne wiązanie [C:\U 
sers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(749): message : zobacz deklarację "closesocket" [C:\Users\booko\Documents\my_gith 
ub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1685,1): error C2375: "connect": zmiana definicji; różne inne wiązanie [C:\Users 
\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(751): message : zobacz deklarację "connect" [C:\Users\booko\Documents\my_github\M
QTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1706,1): error C2375: "ioctlsocket": zmiana definicji; różne inne wiązanie [C:\U 
sers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(756): message : zobacz deklarację "ioctlsocket" [C:\Users\booko\Documents\my_gith 
ub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1729,1): error C2375: "getpeername": zmiana definicji; różne inne wiązanie [C:\U 
sers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(761): message : zobacz deklarację "getpeername" [C:\Users\booko\Documents\my_gith
ub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1750,1): error C2375: "getsockname": zmiana definicji; różne inne wiązanie [C:\U 
sers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(766): message : zobacz deklarację "getsockname" [C:\Users\booko\Documents\my_gith 
ub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1771,1): error C2375: "getsockopt": zmiana definicji; różne inne wiązanie [C:\Us 
ers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(771): message : zobacz deklarację "getsockopt" [C:\Users\booko\Documents\my_githu 
b\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1796,1): error C2375: "htonl": zmiana definicji; różne inne wiązanie [C:\Users\b 
ooko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(778): message : zobacz deklarację "htonl" [C:\Users\booko\Documents\my_github\MQT 
T-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1813,1): error C2375: "htons": zmiana definicji; różne inne wiązanie [C:\Users\b 
ooko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(780): message : zobacz deklarację "htons" [C:\Users\booko\Documents\my_github\MQT 
T-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1831,1): error C2375: "inet_addr": zmiana definicji; różne inne wiązanie [C:\Use 
rs\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(782): message : zobacz deklarację "inet_addr" [C:\Users\booko\Documents\my_github 
\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1849,1): error C2375: "inet_ntoa": zmiana definicji; różne inne wiązanie [C:\Use 
rs\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(784): message : zobacz deklarację "inet_ntoa" [C:\Users\booko\Documents\my_github 
\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1949,1): error C2375: "listen": zmiana definicji; różne inne wiązanie [C:\Users\ 
booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(786): message : zobacz deklarację "listen" [C:\Users\booko\Documents\my_github\MQ 
TT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1968,1): error C2375: "ntohl": zmiana definicji; różne inne wiązanie [C:\Users\b 
ooko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(790): message : zobacz deklarację "ntohl" [C:\Users\booko\Documents\my_github\MQT 
T-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1985,1): error C2375: "ntohs": zmiana definicji; różne inne wiązanie [C:\Users\b 
ooko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(792): message : zobacz deklarację "ntohs" [C:\Users\booko\Documents\my_github\MQT 
T-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2002,1): error C2375: "recv": zmiana definicji; różne inne wiązanie [C:\Users\bo 
oko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(794): message : zobacz deklarację "recv" [C:\Users\booko\Documents\my_github\MQTT 
-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2025,1): error C2375: "recvfrom": zmiana definicji; różne inne wiązanie [C:\User 
s\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(800): message : zobacz deklarację "recvfrom" [C:\Users\booko\Documents\my_github\ 
MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2052,1): error C2375: "select": zmiana definicji; różne inne wiązanie [C:\Users\ 
booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(808): message : zobacz deklarację "select" [C:\Users\booko\Documents\my_github\MQ 
TT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2077,1): error C2375: "send": zmiana definicji; różne inne wiązanie [C:\Users\bo
oko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(815): message : zobacz deklarację "send" [C:\Users\booko\Documents\my_github\MQTT 
-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2100,1): error C2375: "sendto": zmiana definicji; różne inne wiązanie [C:\Users\ 
booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(821): message : zobacz deklarację "sendto" [C:\Users\booko\Documents\my_github\MQ 
TT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2127,1): error C2375: "setsockopt": zmiana definicji; różne inne wiązanie [C:\Us
ers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(829): message : zobacz deklarację "setsockopt" [C:\Users\booko\Documents\my_githu 
b\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2152,1): error C2375: "shutdown": zmiana definicji; różne inne wiązanie [C:\User
s\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(836): message : zobacz deklarację "shutdown" [C:\Users\booko\Documents\my_github\ 
MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2172,1): error C2375: "socket": zmiana definicji; różne inne wiązanie [C:\Users\ 
booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(840): message : zobacz deklarację "socket" [C:\Users\booko\Documents\my_github\MQ 
TT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2197,1): error C2375: "gethostbyaddr": zmiana definicji; różne inne wiązanie [C: 
\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(847): message : zobacz deklarację "gethostbyaddr" [C:\Users\booko\Documents\my_gi 
thub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2219,1): error C2375: "gethostbyname": zmiana definicji; różne inne wiązanie [C: 
\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(852): message : zobacz deklarację "gethostbyname" [C:\Users\booko\Documents\my_gi 
thub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2236,1): error C2375: "gethostname": zmiana definicji; różne inne wiązanie [C:\U 
sers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(854): message : zobacz deklarację "gethostname" [C:\Users\booko\Documents\my_gith 
ub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2276,1): error C2375: "getservbyport": zmiana definicji; różne inne wiązanie [C: 
\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(858): message : zobacz deklarację "getservbyport" [C:\Users\booko\Documents\my_gi 
thub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2295,1): error C2375: "getservbyname": zmiana definicji; różne inne wiązanie [C: 
\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(862): message : zobacz deklarację "getservbyname" [C:\Users\booko\Documents\my_gi 
thub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2314,1): error C2375: "getprotobynumber": zmiana definicji; różne inne wiązanie  
[C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(866): message : zobacz deklarację "getprotobynumber" [C:\Users\booko\Documents\my 
_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2331,1): error C2375: "getprotobyname": zmiana definicji; różne inne wiązanie [C 
:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(868): message : zobacz deklarację "getprotobyname" [C:\Users\booko\Documents\my_g 
ithub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2351,1): error C2375: "WSAStartup": zmiana definicji; różne inne wiązanie [C:\Us 
ers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(872): message : zobacz deklarację "WSAStartup" [C:\Users\booko\Documents\my_githu 
b\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2371,1): error C2375: "WSACleanup": zmiana definicji; różne inne wiązanie [C:\Us 
ers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(876): message : zobacz deklarację "WSACleanup" [C:\Users\booko\Documents\my_githu 
b\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2388,1): error C2375: "WSASetLastError": zmiana definicji; różne inne wiązanie [ 
C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(878): message : zobacz deklarację "WSASetLastError" [C:\Users\booko\Documents\my_ 
github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2405,1): error C2375: "WSAGetLastError": zmiana definicji; różne inne wiązanie [ 
C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(880): message : zobacz deklarację "WSAGetLastError" [C:\Users\booko\Documents\my_ 
github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2426,1): error C2375: "WSAIsBlocking": zmiana definicji; różne inne wiązanie [C: 
\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(882): message : zobacz deklarację "WSAIsBlocking" [C:\Users\booko\Documents\my_gi 
thub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2444,1): error C2375: "WSAUnhookBlockingHook": zmiana definicji; różne inne wiąz 
anie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(884): message : zobacz deklarację "WSAUnhookBlockingHook" [C:\Users\booko\Documen
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2462,1): error C2375: "WSASetBlockingHook": zmiana definicji; różne inne wiązani 
e [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(886): message : zobacz deklarację "WSASetBlockingHook" [C:\Users\booko\Documents\ 
my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2480,1): error C2375: "WSACancelBlockingCall": zmiana definicji; różne inne wiąz 
anie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(888): message : zobacz deklarację "WSACancelBlockingCall" [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2498,1): error C2375: "WSAAsyncGetServByName": zmiana definicji; różne inne wiąz 
anie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(890): message : zobacz deklarację "WSAAsyncGetServByName" [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2526,1): error C2375: "WSAAsyncGetServByPort": zmiana definicji; różne inne wiąz 
anie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(898): message : zobacz deklarację "WSAAsyncGetServByPort" [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2554,1): error C2375: "WSAAsyncGetProtoByName": zmiana definicji; różne inne wią 
zanie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(906): message : zobacz deklarację "WSAAsyncGetProtoByName" [C:\Users\booko\Docume 
nts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2580,1): error C2375: "WSAAsyncGetProtoByNumber": zmiana definicji; różne inne w 
iązanie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(913): message : zobacz deklarację "WSAAsyncGetProtoByNumber" [C:\Users\booko\Docu 
ments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2606,1): error C2375: "WSAAsyncGetHostByName": zmiana definicji; różne inne wiąz 
anie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(920): message : zobacz deklarację "WSAAsyncGetHostByName" [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2632,1): error C2375: "WSAAsyncGetHostByAddr": zmiana definicji; różne inne wiąz 
anie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(927): message : zobacz deklarację "WSAAsyncGetHostByAddr" [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2662,1): error C2375: "WSACancelAsyncRequest": zmiana definicji; różne inne wiąz 
anie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(936): message : zobacz deklarację "WSACancelAsyncRequest" [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2680,1): error C2375: "WSAAsyncSelect": zmiana definicji; różne inne wiązanie [C 
:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(938): message : zobacz deklarację "WSAAsyncSelect" [C:\Users\booko\Documents\my_g 
ithub\MQTT-C\build\mqttc.vcxproj]
  mqtt.c
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(103,1): warning C4005: "AF_IPX": ponowna definicja makra [C:\Users\booko\Docum
ents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(457): message : zobacz poprzednią definicję "AF_IPX" [C:\Users\booko\Documents\my 
_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(147,1): warning C4005: "AF_MAX": ponowna definicja makra [C:\Users\booko\Docum 
ents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(476): message : zobacz poprzednią definicję "AF_MAX" [C:\Users\booko\Documents\my 
_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(185,1): warning C4005: "SO_DONTLINGER": ponowna definicja makra [C:\Users\book 
o\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(399): message : zobacz poprzednią definicję "SO_DONTLINGER" [C:\Users\booko\Docum 
ents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(238,25): error C2011: "sockaddr'': "struct" wpisz ponownie definicję [C:\Users 
\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(482): message : zobacz deklarację "sockaddr" [C:\Users\booko\Documents\my_github\ 
MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(440,5): error C2059: błąd składniowy: "stała" [C:\Users\booko\Documents\my_git 
hub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(575,1): warning C4005: "IN_CLASSA": ponowna definicja makra [C:\Users\booko\Do 
cuments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(284): message : zobacz poprzednią definicję "IN_CLASSA" [C:\Users\booko\Documents 
\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(581,1): warning C4005: "IN_CLASSB": ponowna definicja makra [C:\Users\booko\Do 
cuments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(290): message : zobacz poprzednią definicję "IN_CLASSB" [C:\Users\booko\Documents 
\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(587,1): warning C4005: "IN_CLASSC": ponowna definicja makra [C:\Users\booko\Do 
cuments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(296): message : zobacz poprzednią definicję "IN_CLASSC" [C:\Users\booko\Documents 
\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(598,1): warning C4005: "INADDR_ANY": ponowna definicja makra [C:\Users\booko\D 
ocuments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(301): message : zobacz poprzednią definicję "INADDR_ANY" [C:\Users\booko\Document 
s\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(600,1): warning C4005: "INADDR_BROADCAST": ponowna definicja makra [C:\Users\b 
ooko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(303): message : zobacz poprzednią definicję "INADDR_BROADCAST" [C:\Users\booko\Do
cuments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\shared\ws2def.h(636,28): error C2011: "sockaddr_in'': "struct" wpisz ponownie definicję [C:\Us 
ers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(309): message : zobacz deklarację "sockaddr_in" [C:\Users\booko\Documents\my_gith 
ub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(136,23): error C2011: "fd_set'': "struct" wpisz ponownie definicję [C:\Users\boo 
ko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(65): message : zobacz deklarację "fd_set" [C:\Users\booko\Documents\my_github\MQT 
T-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(156,1): warning C4005: "FD_CLR": ponowna definicja makra [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(94): message : zobacz poprzednią definicję "FD_CLR" [C:\Users\booko\Documents\my_ 
github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(171,1): warning C4005: "FD_SET": ponowna definicja makra [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(99): message : zobacz poprzednią definicję "FD_SET" [C:\Users\booko\Documents\my_ 
github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(180,16): error C2011: "timeval'': "struct" wpisz ponownie definicję [C:\Users\bo 
oko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(108): message : zobacz deklarację "timeval" [C:\Users\booko\Documents\my_github\M 
QTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(236,17): error C2011: "hostent'': "struct" wpisz ponownie definicję [C:\Users\bo 
oko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(164): message : zobacz deklarację "hostent" [C:\Users\booko\Documents\my_github\M 
QTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(249,16): error C2011: "netent'': "struct" wpisz ponownie definicję [C:\Users\boo 
ko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(177): message : zobacz deklarację "netent" [C:\Users\booko\Documents\my_github\MQ 
TT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(256,17): error C2011: "servent'': "struct" wpisz ponownie definicję [C:\Users\bo 
oko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(184): message : zobacz deklarację "servent" [C:\Users\booko\Documents\my_github\M 
QTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(268,18): error C2011: "protoent'': "struct" wpisz ponownie definicję [C:\Users\b 
ooko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(196): message : zobacz deklarację "protoent" [C:\Users\booko\Documents\my_github\ 
MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(364,24): error C2011: "WSAData'': "struct" wpisz ponownie definicję [C:\Users\bo 
oko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(319): message : zobacz deklarację "WSAData" [C:\Users\booko\Documents\my_github\M 
QTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(462,18): error C2011: "sockproto'': "struct" wpisz ponownie definicję [C:\Users\ 
booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(491): message : zobacz deklarację "sockproto" [C:\Users\booko\Documents\my_github 
\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(504,16): error C2011: "linger'': "struct" wpisz ponownie definicję [C:\Users\boo
ko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(528): message : zobacz deklarację "linger" [C:\Users\booko\Documents\my_github\MQ 
TT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(517,1): warning C4005: "SOMAXCONN": ponowna definicja makra [C:\Users\booko\Docu 
ments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(541): message : zobacz poprzednią definicję "SOMAXCONN" [C:\Users\booko\Documents 
\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(552,1): warning C4005: "FD_READ": ponowna definicja makra [C:\Users\booko\Docume 
nts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(559): message : zobacz poprzednią definicję "FD_READ" [C:\Users\booko\Documents\m 
y_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(555,1): warning C4005: "FD_WRITE": ponowna definicja makra [C:\Users\booko\Docum 
ents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(560): message : zobacz poprzednią definicję "FD_WRITE" [C:\Users\booko\Documents\ 
my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(558,1): warning C4005: "FD_OOB": ponowna definicja makra [C:\Users\booko\Documen
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(561): message : zobacz poprzednią definicję "FD_OOB" [C:\Users\booko\Documents\my 
_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(561,1): warning C4005: "FD_ACCEPT": ponowna definicja makra [C:\Users\booko\Docu 
ments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(562): message : zobacz poprzednią definicję "FD_ACCEPT" [C:\Users\booko\Documents 
\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(564,1): warning C4005: "FD_CONNECT": ponowna definicja makra [C:\Users\booko\Doc 
uments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(563): message : zobacz poprzednią definicję "FD_CONNECT" [C:\Users\booko\Document 
s\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(567,1): warning C4005: "FD_CLOSE": ponowna definicja makra [C:\Users\booko\Docum 
ents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(564): message : zobacz poprzednią definicję "FD_CLOSE" [C:\Users\booko\Documents\ 
my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1625,1): error C2375: "accept": zmiana definicji; różne inne wiązanie [C:\Users\ 
booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(739): message : zobacz deklarację "accept" [C:\Users\booko\Documents\my_github\MQ 
TT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1647,1): error C2375: "bind": zmiana definicji; różne inne wiązanie [C:\Users\bo 
oko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(744): message : zobacz deklarację "bind" [C:\Users\booko\Documents\my_github\MQTT 
-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1668,1): error C2375: "closesocket": zmiana definicji; różne inne wiązanie [C:\U 
sers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(749): message : zobacz deklarację "closesocket" [C:\Users\booko\Documents\my_gith 
ub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1685,1): error C2375: "connect": zmiana definicji; różne inne wiązanie [C:\Users 
\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(751): message : zobacz deklarację "connect" [C:\Users\booko\Documents\my_github\M 
QTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1706,1): error C2375: "ioctlsocket": zmiana definicji; różne inne wiązanie [C:\U 
sers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(756): message : zobacz deklarację "ioctlsocket" [C:\Users\booko\Documents\my_gith 
ub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1729,1): error C2375: "getpeername": zmiana definicji; różne inne wiązanie [C:\U 
sers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(761): message : zobacz deklarację "getpeername" [C:\Users\booko\Documents\my_gith 
ub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1750,1): error C2375: "getsockname": zmiana definicji; różne inne wiązanie [C:\U 
sers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(766): message : zobacz deklarację "getsockname" [C:\Users\booko\Documents\my_gith 
ub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1771,1): error C2375: "getsockopt": zmiana definicji; różne inne wiązanie [C:\Us 
ers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(771): message : zobacz deklarację "getsockopt" [C:\Users\booko\Documents\my_githu 
b\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1796,1): error C2375: "htonl": zmiana definicji; różne inne wiązanie [C:\Users\b 
ooko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(778): message : zobacz deklarację "htonl" [C:\Users\booko\Documents\my_github\MQT 
T-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1813,1): error C2375: "htons": zmiana definicji; różne inne wiązanie [C:\Users\b 
ooko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(780): message : zobacz deklarację "htons" [C:\Users\booko\Documents\my_github\MQT 
T-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1831,1): error C2375: "inet_addr": zmiana definicji; różne inne wiązanie [C:\Use 
rs\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(782): message : zobacz deklarację "inet_addr" [C:\Users\booko\Documents\my_github
\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1849,1): error C2375: "inet_ntoa": zmiana definicji; różne inne wiązanie [C:\Use 
rs\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(784): message : zobacz deklarację "inet_ntoa" [C:\Users\booko\Documents\my_github 
\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1949,1): error C2375: "listen": zmiana definicji; różne inne wiązanie [C:\Users\ 
booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(786): message : zobacz deklarację "listen" [C:\Users\booko\Documents\my_github\MQ 
TT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1968,1): error C2375: "ntohl": zmiana definicji; różne inne wiązanie [C:\Users\b 
ooko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(790): message : zobacz deklarację "ntohl" [C:\Users\booko\Documents\my_github\MQT 
T-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(1985,1): error C2375: "ntohs": zmiana definicji; różne inne wiązanie [C:\Users\b 
ooko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(792): message : zobacz deklarację "ntohs" [C:\Users\booko\Documents\my_github\MQT 
T-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2002,1): error C2375: "recv": zmiana definicji; różne inne wiązanie [C:\Users\bo 
oko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(794): message : zobacz deklarację "recv" [C:\Users\booko\Documents\my_github\MQTT
-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2025,1): error C2375: "recvfrom": zmiana definicji; różne inne wiązanie [C:\User 
s\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(800): message : zobacz deklarację "recvfrom" [C:\Users\booko\Documents\my_github\ 
MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2052,1): error C2375: "select": zmiana definicji; różne inne wiązanie [C:\Users\ 
booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(808): message : zobacz deklarację "select" [C:\Users\booko\Documents\my_github\MQ 
TT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2077,1): error C2375: "send": zmiana definicji; różne inne wiązanie [C:\Users\bo 
oko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(815): message : zobacz deklarację "send" [C:\Users\booko\Documents\my_github\MQTT 
-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2100,1): error C2375: "sendto": zmiana definicji; różne inne wiązanie [C:\Users\ 
booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(821): message : zobacz deklarację "sendto" [C:\Users\booko\Documents\my_github\MQ 
TT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2127,1): error C2375: "setsockopt": zmiana definicji; różne inne wiązanie [C:\Us 
ers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(829): message : zobacz deklarację "setsockopt" [C:\Users\booko\Documents\my_githu 
b\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2152,1): error C2375: "shutdown": zmiana definicji; różne inne wiązanie [C:\User 
s\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(836): message : zobacz deklarację "shutdown" [C:\Users\booko\Documents\my_github\ 
MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2172,1): error C2375: "socket": zmiana definicji; różne inne wiązanie [C:\Users\ 
booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(840): message : zobacz deklarację "socket" [C:\Users\booko\Documents\my_github\MQ 
TT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2197,1): error C2375: "gethostbyaddr": zmiana definicji; różne inne wiązanie [C: 
\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(847): message : zobacz deklarację "gethostbyaddr" [C:\Users\booko\Documents\my_gi 
thub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2219,1): error C2375: "gethostbyname": zmiana definicji; różne inne wiązanie [C: 
\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(852): message : zobacz deklarację "gethostbyname" [C:\Users\booko\Documents\my_gi 
thub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2236,1): error C2375: "gethostname": zmiana definicji; różne inne wiązanie [C:\U 
sers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(854): message : zobacz deklarację "gethostname" [C:\Users\booko\Documents\my_gith 
ub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2276,1): error C2375: "getservbyport": zmiana definicji; różne inne wiązanie [C: 
\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(858): message : zobacz deklarację "getservbyport" [C:\Users\booko\Documents\my_gi 
thub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2295,1): error C2375: "getservbyname": zmiana definicji; różne inne wiązanie [C: 
\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(862): message : zobacz deklarację "getservbyname" [C:\Users\booko\Documents\my_gi 
thub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2314,1): error C2375: "getprotobynumber": zmiana definicji; różne inne wiązanie  
[C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(866): message : zobacz deklarację "getprotobynumber" [C:\Users\booko\Documents\my 
_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2331,1): error C2375: "getprotobyname": zmiana definicji; różne inne wiązanie [C 
:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(868): message : zobacz deklarację "getprotobyname" [C:\Users\booko\Documents\my_g 
ithub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2351,1): error C2375: "WSAStartup": zmiana definicji; różne inne wiązanie [C:\Us 
ers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(872): message : zobacz deklarację "WSAStartup" [C:\Users\booko\Documents\my_githu 
b\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2371,1): error C2375: "WSACleanup": zmiana definicji; różne inne wiązanie [C:\Us 
ers\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(876): message : zobacz deklarację "WSACleanup" [C:\Users\booko\Documents\my_githu 
b\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2388,1): error C2375: "WSASetLastError": zmiana definicji; różne inne wiązanie [ 
C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(878): message : zobacz deklarację "WSASetLastError" [C:\Users\booko\Documents\my_ 
github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2405,1): error C2375: "WSAGetLastError": zmiana definicji; różne inne wiązanie [ 
C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(880): message : zobacz deklarację "WSAGetLastError" [C:\Users\booko\Documents\my_ 
github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2426,1): error C2375: "WSAIsBlocking": zmiana definicji; różne inne wiązanie [C: 
\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(882): message : zobacz deklarację "WSAIsBlocking" [C:\Users\booko\Documents\my_gi 
thub\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2444,1): error C2375: "WSAUnhookBlockingHook": zmiana definicji; różne inne wiąz 
anie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(884): message : zobacz deklarację "WSAUnhookBlockingHook" [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2462,1): error C2375: "WSASetBlockingHook": zmiana definicji; różne inne wiązani 
e [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(886): message : zobacz deklarację "WSASetBlockingHook" [C:\Users\booko\Documents\ 
my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2480,1): error C2375: "WSACancelBlockingCall": zmiana definicji; różne inne wiąz 
anie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(888): message : zobacz deklarację "WSACancelBlockingCall" [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2498,1): error C2375: "WSAAsyncGetServByName": zmiana definicji; różne inne wiąz 
anie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(890): message : zobacz deklarację "WSAAsyncGetServByName" [C:\Users\booko\Documen
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2526,1): error C2375: "WSAAsyncGetServByPort": zmiana definicji; różne inne wiąz
anie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(898): message : zobacz deklarację "WSAAsyncGetServByPort" [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2554,1): error C2375: "WSAAsyncGetProtoByName": zmiana definicji; różne inne wią 
zanie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(906): message : zobacz deklarację "WSAAsyncGetProtoByName" [C:\Users\booko\Docume 
nts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2580,1): error C2375: "WSAAsyncGetProtoByNumber": zmiana definicji; różne inne w 
iązanie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(913): message : zobacz deklarację "WSAAsyncGetProtoByNumber" [C:\Users\booko\Docu 
ments\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2606,1): error C2375: "WSAAsyncGetHostByName": zmiana definicji; różne inne wiąz
anie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(920): message : zobacz deklarację "WSAAsyncGetHostByName" [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2632,1): error C2375: "WSAAsyncGetHostByAddr": zmiana definicji; różne inne wiąz 
anie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(927): message : zobacz deklarację "WSAAsyncGetHostByAddr" [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2662,1): error C2375: "WSACancelAsyncRequest": zmiana definicji; różne inne wiąz 
anie [C:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(936): message : zobacz deklarację "WSACancelAsyncRequest" [C:\Users\booko\Documen 
ts\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock2.h(2680,1): error C2375: "WSAAsyncSelect": zmiana definicji; różne inne wiązanie [C
:\Users\booko\Documents\my_github\MQTT-C\build\mqttc.vcxproj]
F:\Windows Kits\10\Include\10.0.18362.0\um\winsock.h(938): message : zobacz deklarację "WSAAsyncSelect" [C:\Users\booko\Documents\my_g 
ithub\MQTT-C\build\mqttc.vcxproj]
```
</details>

Sorry for my polish locales, I am not a pro CMake user and never faces something like that.
I have built paho-mqtt-c without much issues using Cmake though.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,13 +11,13 @@ option(MQTT_C_TESTS "Build MQTT-C tests?" OFF)
 list (APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
 # MQTT-C library
-add_library(mqttc STATIC 
+add_library(mqttc STATIC
     src/mqtt_pal.c
     src/mqtt.c
 )
 target_include_directories(mqttc PUBLIC include)
 target_link_libraries(mqttc PUBLIC 
-    $<$<C_COMPILER_ID:MSVS>:ws2_32>
+    $<$<C_COMPILER_ID:MSVC>:ws2_32>
 )
 
 if(MQTT_C_BearSSL_SUPPORT)
@@ -89,7 +89,7 @@ if(MQTT_C_EXAMPLES)
 endif()
 
 # Build tests
-if(MQTT_C_TESTS)    
+if(MQTT_C_TESTS)
     find_path(CMOCKA_INCLUDE_DIR cmocka.h)
     find_library(CMOCKA_LIBRARY cmocka)
     if((NOT CMOCKA_INCLUDE_DIR) OR (NOT CMOCKA_LIBRARY))

--- a/include/mqtt_pal.h
+++ b/include/mqtt_pal.h
@@ -117,10 +117,10 @@ extern "C" {
     #endif
 #elif defined(_MSC_VER)
     #include <limits.h>
+    #include <winsock2.h>
     #include <windows.h>
     #include <time.h>
     #include <stdint.h>
-    #include <winsock2.h>
 
     typedef SSIZE_T ssize_t;
     #define MQTT_PAL_HTONS(s) htons(s)

--- a/src/mqtt_pal.c
+++ b/src/mqtt_pal.c
@@ -223,7 +223,7 @@ ssize_t mqtt_pal_recvall(mqtt_pal_socket_handle fd, void* buf, size_t bufsz, int
 ssize_t mqtt_pal_sendall(mqtt_pal_socket_handle fd, const void* buf, size_t len, int flags) {
     size_t sent = 0;
     while(sent < len) {
-        int tmp = BIO_write(fd, buf + sent, len - sent);
+        int tmp = BIO_write(fd, (const char*)buf + sent, len - sent);
         if (tmp > 0) {
             sent += (size_t) tmp;
         } else if (tmp <= 0 && !BIO_should_retry(fd)) {
@@ -235,13 +235,14 @@ ssize_t mqtt_pal_sendall(mqtt_pal_socket_handle fd, const void* buf, size_t len,
 }
 
 ssize_t mqtt_pal_recvall(mqtt_pal_socket_handle fd, void* buf, size_t bufsz, int flags) {
-    const void *const start = buf;
+    const char* const start = buf;
+    char* bufptr = buf;
     int rv;
     do {
-        rv = BIO_read(fd, buf, bufsz);
+        rv = BIO_read(fd, bufptr, bufsz);
         if (rv > 0) {
             /* successfully read bytes from the socket */
-            buf += rv;
+            bufptr += rv;
             bufsz -= rv;
         } else if (!BIO_should_retry(fd)) {
             /* an error occurred that wasn't "nothing to read". */
@@ -249,7 +250,7 @@ ssize_t mqtt_pal_recvall(mqtt_pal_socket_handle fd, void* buf, size_t bufsz, int
         }
     } while (!BIO_should_read(fd));
 
-    return (ssize_t)(buf - start);
+    return (ssize_t)(bufptr - start);
 }
 
 #elif defined(__unix__) || defined(__APPLE__)

